vault/static/vault/css/metadata/form.css

```diff
@@ -22,3 +22,7 @@
 #metadata-form .d-flex, #metadata-form  .col {
     padding: 0 15px;
 }
+
+.metadata-card {
+    background-color: rgba(var(--bs-body-bg-rgb), 0.9);
```

Comment:
I wonder why alpha=0.9 in this context?

Reply:
It was my design choice. Here are some examples with different alpha values.

Alpha=1
<img width="1316" alt="image" src="https://github.com/user-attachments/assets/d4290063-5299-443e-ac1a-2a50bfdce525">

Alpha=0
<img width="1316" alt="image" src="https://github.com/user-attachments/assets/da9efab0-c6a4-4934-b5bb-0ff6abc59954">

Alpha=0.5
<img width="1316" alt="image" src="https://github.com/user-attachments/assets/f239e714-310b-4984-b7b7-d850a5f2c3ff">

Alpha=0.9
<img width="1316" alt="image" src="https://github.com/user-attachments/assets/62b170cb-76d5-4e5d-ad9c-be785553f13f">